moo.js

```diff
@@ -140,6 +141,22 @@
     return result
   }
 
+  function isKeyword(literal, otherRules) {
```

Comment:
I wouldn't call this `isKeyword` since it doesn't return a boolean, it returns a rule.